docs/docs/chitchat-faqs.mdx

```diff
@@ -4,258 +4,78 @@ sidebar_label: Chitchat and FAQs
 title: Chitchat and FAQs
 ---
 
-import useBaseUrl from '@docusaurus/useBaseUrl';
+FAQs and chitchat both involve non-contextual questions with single-message answers. 
+For example, in the following conversation, every question's answer is independent of context and could
+just as easily be answered on its own:
 
-<!-- TODO: Restructure all conversations patterns pages to not depend on one another -->
 
-After following the basics of [prototyping an assistant](./prototype-an-assistant.mdx), we'll
-now walk through building a basic FAQ chatbot and then build a bot that can handle
-contextual conversations.
+> User: What's your name
+>
+>> Bot: My name is Sara!
+>
+> User: Which languages can I build assistants in?
+> 
+>> Bot: You can use Rasa to build assistants in any language you want!
+> 
+> User: What’s Rasa X?
+>
+>> Bot: Rasa X is a tool to learn from real conversations and improve your assistant. Read more [here](https://rasa.com/docs/rasa-x/)
 
-<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="build-faq-assistant"></a>
 
-FAQ assistants are the simplest assistants to build and a good place to get started.
-These assistants allow the user to ask a simple question and get a response. We're going to
-build a basic FAQ assistant using features of Rasa designed specifically for this type of assistant.
 
-In this section we're going to cover the following topics:
+This page is a guide on the concepts and training data you need to handle non-contextual questions like
+FAQ and chitchat.
+You can use the example configuration and training data on this page to build an assistant
+that can handle a conversation like the one above, 
+or you can apply the guidelines to your own use case.
 
-* [Responding to simple intents](./chitchat-faqs.mdx#respond-with-memoization-policy) with the MemoizationPolicy
+## Concepts
 
-* [Handling FAQs](./chitchat-faqs.mdx#faqs-response-selector) using the ResponseSelector
+To handle FAQ and chitchat you'll need a rule-based dialogue management policy ([the `RulePolicy`](#rulepolicy)), and an easy way
```

Comment:
also nit-picky :) no comma after RulePolicy

I also see a few places where we use FAQ when it should be plural (FAQs)